### ISSUE TITLE
Speed metric

### ISSUE THREAD

**Issue by Flieh** [Flieh/diplicity-beta-feedback#16]
Martin, you've explained that the player speed measurement only takes in to account whether or not the player ticked the ready box before the deadline. Personally this is not a metric that I care much about but it is one that excites many people who like to play fast turnaround games. Would it be difficult to modify the metric to reflect the actual time between the end of one phase and the ticking if the ready box. Would it be useful? Your approach may be the best and perhaps over time it will reflect the actual turnaround you can expect from a specific player... Not sure though. I like to use up all my time (not always, but often) but I do usually tick the ready box shortly before deadline, so I shouldn't be considered a fast player even if I do always tick the box. See what I mean?

**Comment by zond** [comment]
Yeah, it's not perfect. It's just my way of separating out the "I'm ready" checkboxing from the reliability (where it didn't really belong).
